src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -248,13 +244,20 @@
           client_config: client_config,
           timeout: timeout,
           app_name: app_name,
-          app_version: app_version
+          app_version: app_version,
+          lib_name: lib_name,
+          lib_version: lib_version,
         )
       @end
 
-      google_api_client = "#{app_name}/#{app_version} " @\
-        "#{CODE_GEN_NAME_VERSION} gax/#{Google::Gax::VERSION} " @\
-        "ruby/#{RUBY_VERSION}".freeze
+      if app_name || app_version
+        warn "`app_name` and `app_version` are no longer being used in the request headers."
+      end
+
+      google_api_client = "#{lib_name}/{lib_version} " @\
+        "gax/#{Google::Gax::VERSION} grpc/#{GRPC::VERSION} " @\
+        "gl-ruby/#{RUBY_VERSION}".freeze
```

Comment:
Move `gl-ruby` to the front of the line (based on changes to the spec from yesterday evening).